### ISSUE TITLE
Install a great AV setup for the events room

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#685]
The events room will be used lots for meetups.  In Dinky at present we lose a reasonable amount of seating space to host a table to project from, so it'd be good to fix that.  Ditto having good audio setup and an easy way to hook into it.

We should...
 * [ ] Install a good screen, presumably at the partition wall end of the room
 * [ ] Buy a better projector (one with HDMI input, as more and more presenters don't have VGA capabilities)
 * [ ] Mount the projector on the ceiling, to free up floorspace
 * [ ] Have a place to position your laptop when presenting (I'm not sure this needs to _quite_ be a podium, but something approaching that)
 * [ ] Set up @cheapjack's PA to provide an easy way to play audio from the laptop, and optionally a few microphones (we've got microphones and mic stands, but would need some sort of mixer for them)

What else?  Any recommendations for projector? Or screen?

**Comment by jackie1050** [comment]
I have spoken to @magman2112, who will discuss with the electrician when he sees him. It seems as though this might be quite expensive.

**Comment by ajlennon** [comment]
I'd very much like to link this in with the idea of video-conferencing meetings and live streaming DoES events. Also perhaps in future VR and AR.

**Comment by johnmckerrell** [comment]
The electrician has said that the socket is quite doable, I think he'd use conduit for it which makes things easier/cheaper. He also said he'd run the HDMI in if we wanted but we'd need to provide that. Should we also be running VGA for people who need that too or are we ok with just HDMI these days? Are those two both ok running alongside higher voltage?

**Comment by ajlennon** [comment]
I believe VGA is still needed myself. Also USB-C if that's a possibility as we'll all be going over to that soon I think

**Comment by amcewen** [comment]
This isn't needed before we move in, so dropping the `DoES Move: The Tapestry` label

**Comment by amcewen** [comment]
We've got a rackmount amp and mixer now, which should let us sort out some decent audio.  We need to check we've got the right cables, etc. to hook it all up.  I think in the short term at least it'll be easiest to pair the mixer with Ross' PA (as we don't have decent speakers for the amp)

**Comment by cheapjack** [comment]
Yep the indidually powered speakers both take 1/4" jack or XLR inputs from the mixer, worth checking they are set to line level and have the gain controller set equally (next door to the power switch, ins and Outs) before we use it next :)

**Comment by amcewen** [comment]
The mixer and amp have been mounted in the small server cabinet and along with the PA are now set up in the events space.  I've written some simple instructions on use on the top of the server cabinet.
[!](https://pbs.twimg.com/media/Df6ne71XUAISMgX.jpg)

I've ordered some [extension leads](http://cpc.farnell.com/pro-signal/psg00741/xlr-socket-to-plug-3m/dp/AV14227?st=XLR%20Lead) and [XLR to 1/4" jack adapters](http://cpc.farnell.com/pro-signal/psg01405/lead-jack-skt-to-xlr-plug-0-2m/dp/AV14573) so we can hook up the three microphones to the set-up.  

The AV cabinet had the locks drilled out to get into it, so I've ordered [two replacement 20mm locks](http://www.ebay.co.uk/itm/152505075459) too.

There are also 4 mic stands, which are in a box in the corner of the events space, next to the AV cabinet.

**Comment by amcewen** [comment]
The extension leads, adapters and replacement locks all arrived today.  Now need fitting.

**Comment by magman2112** [comment]
The new locks for the AV cabinet are installed and working. Also the extension leads and adaptors have been installed and tested and are all working. 

I've use the twist ties that came with the leads to keep the leads reasonably tidy for now, but it would be good to get some Velcro ties to help avoid a rat's nest of cables in the future.

**Comment by goatchurchprime** [comment]
Wishlist contents moved from the AV page in the wiki to here where the discussion is taking place.

-------------------
# Proposed Audio Visual Setup

# At a minimum
 * Projector mount to attach on to the beam
 * Long VGA cable to "presentation point" at front of room.
 * Long power cable to projector
 * CAT 5 cable from hub to "presentation point" so people can get more reliable network connection.

To Discuss: What screen to use?
Can we project onto the large whiteboard that's in the conference room/storage, or another whiteboard? Would then be dual use and save space of a screen.

Presumably we could try this one day?  I suspect it'd be too reflective, but it shouldn't take long to test - AdrianMcEwen

We could try making our own wall-mounted screen, by painting a sheet of MDF with [Black widow paint mix](http://en.wikipedia.org/wiki/Black_Widow_%28paint_mix%29#Black_Widow_EU) - AdrianMcEwen

To Discuss: Will we have issues with length of VGA Cable?
Given long cables can be prone to degrading signal, do we need to consider options for boosting or improving the setup?

Again, can we test with the existing cable?  See if it'll reach, or how much further it would need to reach? - AdrianMcEwen

## If we have the budget
 * VGA converter for modern Macbook display outputs
 * If existing whiteboard is unsuitable, a flat screen/surface on the wall (We gain a foot of extra throw distance for projector, space for audience and easier to setup)
 * Speakers mounted on wall, with audio jack at "presentation point"

## Dream setup
 * UHD Projector
 * HD Webcam + mics for streaming/recording presentations

## Echo Reduction
A number of people have commented on the echo on the room - We should consider ways to reduce this, perhaps:
 * Projector Screen could help
 * Carpeted tiles on ceiling
 * Soft coverings on chairs
 * Plants around the room

**Comment by johnmckerrell** [comment]
Don't forget #569 (hearing loop)

**Comment by amcewen** [comment]
Having recently watched [this interesting video from last-year's FOSDEM](https://archive.fosdem.org/2025/schedule/event/fosdem-2025-6832-fosdem-videobox-2025/), I figured I'd at least make a note (which is why I'm sticking it as a comment on this issue rather than spawning a full "we should do this" issue just for it to languish till we've got funds/inclination) of [the FOSDEM videobox repo](https://github.com/fosdem/video) because it would be lovely to have one of these mounted in the AV rack to make streaming talks simpler.

On the talk they said it was ~Eur450 in parts (and it has a handful of custom PCBs, which might be more expensive in one-off fabbing; but we might be able to buy some from them?)
